docs/downstream/odh-install-crc.md

```diff
@@ -0,0 +1,83 @@
+# Installing ODH using ArgoCD
+
+## Preparation
+
+First make sure that you have the OpenDataHub operator available in your OpenShift.
+
+```bash
+oc get packagemanifests -n openshift-marketplace | grep opendatahub-operator
+```
+
+If it's not present, then you need to troubleshoot the marketplace: https://github.com/operator-framework/operator-marketplace/issues/344
```

Comment:
Can we convert these links to hyperlinks as well  in the form [ text ] ( link )

Reply:
Same as above. I suggest doing it for relative links and keeping the external ones.